src/marshmallow/validate.py

```diff
@@ -170,7 +170,7 @@ def __call__(
     _regex = RegexMemoizer()
 
     default_message = "Not a valid URL."
-    default_schemes = {"http", "https", "ftp", "ftps"}
+    default_schemes = {"http", "https", "ftp", "ftps", "file"}
```

Comment:
i don't think we want to change the default accepted schems, as this could be considered a breaking change.